### PR TITLE
Improvements to installation with Heroku (Fixes issue: #1666)

### PR DESCRIPTION
**Pull request by Beta-King**
This PR adds more information and contributes to the docs folder, INSTALLATION_HEROKU.md for better installation :D  

Fixes issue #1666: Make Docs more informative !!

### CODE REVIEW

**Comment by Beta-King** [comment]
this PR doesnt fix any issue, it just makes the doc more informative :D

**Comment by Beta-King** [comment]
@abishekvashok @samyakbvs @harshit98 @isuruAb --> please review and merge this PR :D

**Comment by Beta-King** [comment]
all five checks have passed, please review and merge this PR :D

File: docs/INSTALLATION_HEROKU.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ You can use the one click deployment:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fossasia/open-event-webapp/tree/development)
 
+If however you live out-side the United States and or Europe you will need to manually Deploy Open Event Web App Generator on Heroku!
```
Comment (abishekvashok) [inline docs/INSTALLATION_HEROKU.md]: I can deploy it when I am outside US and Europe in India using the above link.

**Comment by Beta-King** [comment]
@abishekvashok made some changes, could you approve now ??

File: docs/INSTALLATION_HEROKU.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ You can use the one click deployment:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fossasia/open-event-webapp/tree/development)
 
+If however you live out-side the United States and or Europe you may need to manually Deploy Open Event Web App Generator on Heroku as many face problems through this process. However give it a go.
```
[the remark this answers is not in the document]
Comment (geekyd) [reply docs/INSTALLATION_HEROKU.md]: Only use capital letters for the first letter of new sentence. 

> Open Event Web App Generator on Heroku

[the remark this answers is not in the document]
Comment (Beta-King) [reply docs/INSTALLATION_HEROKU.md]: ohk making changes right now :D

File: docs/INSTALLATION_HEROKU.md [hunk 3]
```diff
@@ -91,6 +94,12 @@ $ heroku open
 It will fire up your deployed app like this:
 ![App running](screenshots/heroku_7.png)
 
+If However, the website does not fire up, don’t worry. There are two things you can try to resolve this…
```
Comment (geekyd) [inline docs/INSTALLATION_HEROKU.md]: Rephrase this 

> However, if the web app does not fire up, try one of the following solution to resolve the issue:

File: docs/INSTALLATION_HEROKU.md [hunk 4]
```diff
@@ -91,6 +94,12 @@ $ heroku open
...
+1. Open a new terminal and write: `$ heroku open`
+2. Go to your heroku account on the web and deploy your app from there
+
+If one doesn’t work then try the other.
```
Comment (geekyd) [inline docs/INSTALLATION_HEROKU.md]: Remove this line

**Comment by Beta-King** [comment]
done @geekyd :D could you approve and merge this now ??

File: docs/INSTALLATION_HEROKU.md [hunk 5]
```diff
@@ -6,6 +6,8 @@ You can use the one click deployment:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fossasia/open-event-webapp/tree/development)
 
+If however you live out-side the United States or Europe you may need to manually deploy Open-Event WebApp Generator on Heroku as many face problems through this process. However give it a go.
```
Comment (geekyd) [inline docs/INSTALLATION_HEROKU.md]: Please change this too
> generator on heroku

Comment (Beta-King) [reply docs/INSTALLATION_HEROKU.md]: ohkie :D

**Comment by Beta-King** [comment]
@geekyd done !! :D

File: docs/INSTALLATION_HEROKU.md (hunk 3 above)
Comment (geekyd) [reply docs/INSTALLATION_HEROKU.md]: @Beta-King check this

Comment (Beta-King) [reply docs/INSTALLATION_HEROKU.md]: ohk wait :D ill do it rn

**Comment by Beta-King** [comment]
@geekyd  made changes :D can we merge now ??

**Comment by Beta-King** [comment]
Can we merge ?? @geekyd

**Comment by Beta-King** [comment]
Making changes now @geekyd

**Comment by Beta-King** [comment]
Wait, ill make a new PR, having trouble with rebasing :D

**Pull request closed without merging**